templates/07-chess/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "near-chess",
+  "description": "",
+  "version": "0.0.1",
+  "scripts": {
+    "build": "gulp",
+    "deploy:pages": "gh-pages -d src",
+    "deploy": "npm run build && near deploy --account_id=near-chess-devnet --node_url=https://studio.nearprotocol.com/devnet && npm run deploy:pages",
```

Comment:
how would this work on devnet when they don't have the key to near-chess-devnet ?